### ISSUE TITLE
[UI] Login in local user page still referring to Dex

### ISSUE THREAD

**Issue by enrichman** [epinio/ui#240]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Below the login with the local user page there is still a link referring to the "Dex login".

![image](https://github.com/epinio/ui/assets/1763949/276b25a2-28e3-452b-aedc-92638999be0b)

### Expected Behavior

It should be "Log in with SSO" or "Use SSO" instead.

### Environment

```markdown
- OS:
- Architecture:
- Epinio Version:
- Cluster:
  - Provider:
  - Options:
  - Kubernetes Version:
```

**Comment by mmartin24** [comment]
Checked in the latest `epinio-dev`
Tests adjusted here: https://github.com/epinio/epinio-end-to-end-tests/pull/345

![image](https://github.com/epinio/ui/assets/37271841/bbbb4d24-100e-4977-8da6-25e47c5efc86)
![image](https://github.com/epinio/ui/assets/37271841/e84b8986-2b1f-4fbf-8916-4d15c018861b)

**Comment by thehejik** [comment]
Verified fixed in epinio v1.9.0-rc3:
![image](https://github.com/epinio/ui/assets/12828077/d5d436df-640e-451f-85ce-0bc9994600e6)
